### ISSUE TITLE
Compile error on make.. make: Fatal error: Command failed for target `mod_auth.o'

### ISSUE THREAD

**Issue by msburbach** [proftpd/proftpd#725]
What I Did

Trying to compile ProFTPD 1.3.6 on Solaris 11.3 and I'm running into a couple of warnings/errors when running make.

Configure command:
```
./configure --enable-ctrls --enable-nls --with-modules=mod_tls:mod_ldap:mod_sftp: --with-includes=/usr/local/include --with-libraries=/usr/local/lib
```
Make error messages:
```
mod_auth.c: In function `login_failed':
mod_auth.c:356: warning: implicit declaration of function `loginfailed'
mod_auth.c:357: error: `AUDIT_FAIL' undeclared (first use in this function)
mod_auth.c:357: error: (Each undeclared identifier is reported only once
mod_auth.c:357: error: for each function it appears in.)
mod_auth.c: In function `login_succeeded':
mod_auth.c:420: warning: implicit declaration of function `loginsuccess'
*** Error code 1
make: Fatal error: Command failed for target `mod_auth.o'
Current working directory /export/home/proftpd-1.3.6/modules
*** Error code 1
make: Fatal error: Command failed for target `modules'
```
What I Expected/Wanted

This is my first attempt at upgrading / recompiling ProFTPD.  Any help is greatly appreciated.

 ProFTPD Version and Configuration

Please help us reproduce the problem/issue you are encountering.  To do this,
we need to know which version of ProFTPD you are using, how it was built,
_etc_.  The following command is an easy way to get all of this information:

This version of ProFTPD was compiled with the same steps I mentioned above.
```
    $ proftpd -V
Compile-time Settings:
  Version: 1.3.5b (maint)
  Platform: SOLARIS2 (SOLARIS2_11) [SunOS 5.11 sun4v]
  Built: Thu Apr 21 2016 10:07:16 MDT
  Built With:
    configure  '--enable-ctrls' '--enable-nls' '--with-modules=mod_tls:mod_ldap:mod_sftp' '--with-includes=/usr/local/ssl/include' '--with-libraries=/usr/local/ssl/lib'

  CFLAGS: -O2 -Wall
  LDFLAGS: -L$(top_srcdir)/lib  -L/usr/local/ssl/lib
  LIBS: -lsendfile -lintl  -lssl -lcrypto -lldap -llber  -lssl  -lpam  -lcrypto -lz -lsupp -lsocket -lnsl -lresolv  -liconv

  Files:
    Configuration File:
      /usr/local/etc/proftpd.conf
    Pid File:
      /usr/local/var/proftpd.pid
    Scoreboard File:
      /usr/local/var/proftpd.scoreboard

  Features:
    - Autoshadow support
    + Controls support
    + curses support
    - Developer support
    - DSO support
    + IPv6 support
    + Largefile support
    - Lastlog support
    - Memcache support
    - ncurses support
    + NLS support
    + OpenSSL support
    - PCRE support
    - POSIX ACL support
    + Shadow file support
    + Sendfile support
    + Trace support

  Tunable Options:
    PR_TUNABLE_BUFFER_SIZE = 1024
    PR_TUNABLE_DEFAULT_RCVBUFSZ = 8192
    PR_TUNABLE_DEFAULT_SNDBUFSZ = 8192
    PR_TUNABLE_GLOBBING_MAX_MATCHES = 100000
    PR_TUNABLE_GLOBBING_MAX_RECURSION = 8
    PR_TUNABLE_HASH_TABLE_SIZE = 40
    PR_TUNABLE_NEW_POOL_SIZE = 512
    PR_TUNABLE_SCOREBOARD_BUFFER_SIZE = 80
    PR_TUNABLE_SCOREBOARD_SCRUB_TIMER = 30
    PR_TUNABLE_SELECT_TIMEOUT = 30
    PR_TUNABLE_TIMEOUTIDENT = 10
    PR_TUNABLE_TIMEOUTIDLE = 600
    PR_TUNABLE_TIMEOUTLINGER = 30
    PR_TUNABLE_TIMEOUTLOGIN = 300
    PR_TUNABLE_TIMEOUTNOXFER = 300
    PR_TUNABLE_TIMEOUTSTALLED = 3600
    PR_TUNABLE_XFER_SCOREBOARD_UPDATES = 10
```
In addition, we need to see *all* of the ProFTPD configuration files you are
using (*minus* any sensitive information like passwords, of course).  Armed
with the version and configuration data, then, we can set up ProFTPD locally
using the same configuration, and see what happens.

[proftpd.zip](https://github.com/proftpd/proftpd/files/2098367/proftpd.zip)

**Comment by msburbach** [comment]
[make.txt](https://github.com/proftpd/proftpd/files/2098481/make.txt)

**Comment by Castaglia** [comment]
I believe this is a duplicate of [Bug#4304](http://bugs.proftpd.org/show_bug.cgi?id=4304); would you be able to try the latest code from the master branch on GitHub, to see if that compiles?

**Comment by msburbach** [comment]
That did the trick!  I also had to update the permissions on glue.sh to make it executable but everything has compiled successfully.  Thank you!
